integration/client-cli/admin-cli/src/main/bin/kcadm.sh

```diff
@@ -20,4 +20,14 @@ if [ "x$RESOLVED_NAME" = "x" ]; then
 fi
 
 DIRNAME=`dirname "$RESOLVED_NAME"`
-java $KC_OPTS -cp $DIRNAME/client/keycloak-admin-cli-${project.version}.jar org.keycloak.client.admin.cli.KcAdmMain "$@"
\ No newline at end of file
+
+
+# Uncomment out these lines if you are integrating with `kcinit`
```

Comment:
Having to modify the kcadmin.sh is not really an elegant solution. Also, this only changes it for Linux and ignores kcadmin for WIndows.